doc/examples/ml/neon/infer.json

```diff
@@ -0,0 +1,38 @@
+{
+  "pipeline": {
+    "name": "inference"
+  },
+  "transform": {
+    "image": "dwhitena/neon-inference",
+    "cmd": [ 
+	"python",
+	"examples/imdb/auto_inference.py",
+	"--model_weights",
+	"/pfs/model/imdb.p",
+	"--vocab_file",
+	"/pfs/model/imdb.vocab",
+	"--review_files",
+	"/pfs/reviews",
+	"--output_dir",
+	"/pfs/out"
+    ]
+  },
+  "parallelism_spec": {
```

Comment:
In pipeline.jpg, there are parallel inference workers. Is something that neon gives us but we don't use pachyderm parallelism?